include/psp2/screenshot.h

```diff
@@ -1,5 +1,5 @@
 /**
- * \usergroup{SceScreenShot}
+ * \usergroup{SceScreenshot}
```

Comment:
@CelesteBlue-dev True, but it was renamed to ScreenShot everywhere else by francar, so that should probably match too, right? 😉